kernel/include/timeout_q.h

```diff
@@ -57,6 +59,7 @@ s32_t z_timeout_remaining(struct _timeout *timeout);
 #define _add_thread_timeout(th, to) do {} while (0 && (void *)to && (void *)th)
 #define _abort_thread_timeout(t) (0)
 #define _get_next_timeout_expiry() (K_FOREVER)
+#define _set_timeout_expiry(t, i) do {} while (0)
```

Comment:
Reminds me that I need to figure out the "right way" to do these noop fallbacks.  Empty function bodies were problematic to MISRA apparently, but I think I might be winning that fight.